source/material_model/visco_plastic.cc

```diff
@@ -310,13 +310,7 @@ namespace aspect
                                                                                       elastic_shear_moduli[j]);
 
               // Step 2b: calculate current (viscous + elastic) stress magnitude
-              current_stress = viscosity_pre_yield * current_edot_ii;
-
-              // The viscoelastic strain rate is divided by 2 here as the Drucker Prager
-              // viscosity calculation below assumes stress = 2 * viscosity * strain_rate_invariant,
-              // whereas the combined viscoelastic + viscous stresses already include the
-              // 2x factor (see computation of edot inside elastic_rheology).
-              current_edot_ii /= 2.;
+              current_stress = 2. * viscosity_pre_yield * current_edot_ii;
```

Comment:
As `current_stress = 2. * viscosity_pre_yield * current_edot_ii` now holds true regardless if viscoelasticity is turned on or off, should we move this line (and the equivalent one above, line 290) out of the if-else statement?

Reply:
Good point. Now changed.